Autopilot/PathManager/Inc/landingManager.hpp

```diff
@@ -0,0 +1,64 @@
+#pragma once
+#include "waypointManager.hpp"
+
+const static int SET_THROTTLE_OFF = -1000;
+const static double DISTANCE_OF_LANDING = 10; //in meters
+const static int METERS_PER_DEG_LAT = 111320;
+const static int ANGLE_OF_LANDING = 5; //in degrees
+const static int STALL_SPEED_NO_PACKAGE = 20; //in km/h
+const static int STALL_SPEED_WITH_PACKAGE = 30; //in km/h
+const static double PI = 3.14159265358979323846;
+const static int FLARE_ALTITUDE = 2; //in meters
+const static double DECRAB_ALTITUDE = 0.7; //in meters
+const static double TOUCHDOWN_ALTITUDE = 0.05; //in meters
```

Comment:
Hmmm, @Fberbari I don't think we should be jamming constants into code but I'm not quite sure how we should implement constants like these.... I was thinking text file but that introduces it's own complications. Thoughts?